src/Server/Helper.php

```diff
@@ -159,7 +158,7 @@ public function parseRequestParams($method, array $bodyParams, array $queryParam
      *
      * @api
```

Comment:
do we know the array k,v type?

Reply:
It is specified.

Generally, I did specify the type in return values when known and left it open in params where it does not matter.